src/main/java/org/folio/list/services/ListService.java

```diff
@@ -77,13 +77,21 @@ public class ListService {
   private final ListVersionRepository listVersionRepository;
   private final ListVersionMapper listVersionMapper;
 
-  public ListSummaryResultsDTO getAllLists(Pageable pageable, List<UUID> ids,
-                                           List<UUID> entityTypeIds, Boolean active, Boolean isPrivate, OffsetDateTime updatedAsOf) {
+  public ListSummaryResultsDTO getAllLists(Pageable pageable, List<UUID> ids, List<UUID> entityTypeIds, Boolean active,
+                                           Boolean isPrivate, Boolean includeDeleted, OffsetDateTime updatedAsOf) {
```

Comment:
Tiny nit/personal opinion: I'd make the new parameter a primitive `boolean` and do the conversion in the controller. The fact that it's optional/nullable seems like something that's more of an API concern, so I'd handle it there